README.md

```diff
@@ -22,6 +22,11 @@ Get your OpenAI API Key [here](https://platform.openai.com/account/api-keys) and
 pip install codeinterpreterapi
 ```
 
+### Using docker
+```bash
+docker-compose up --build
```

Comment:
Why docker-compose? I don't see the need for this for a single container application

Reply:
you are right, but i am about to attempt to use Redis memory so I think maybe someone will need docker-compose to run memory manager components like Redis or SQL,

Reply:
I just push another commit which save chat memory via the session id to json file or redis

Reply:
Yeah but still I think its not the place to for a docker-compose file. Everyone should do that in their own backend setup.

Reply:
It is my habit to use docker-compose to start the system in the most convenient way without remembering docker variables. So in case, you think it is unnecessary, I will remove it in the next commit